docs/specification/draft/basic/enterprise.mdx

```diff
@@ -0,0 +1,294 @@
+| `aud`        | REQUIRED          | The Issuer URL of the MCP server's authorization server.                                                                       | [Section 4.1.3 of RFC7519](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3) |
+| `client_id`  | REQUIRED          | An identifier of the MCP client that this JWT was issued to; SHOULD be a client_id as defined in Section 4.3 of RFC8693.       | [Section 4.3 of RFC8693](https://datatracker.ietf.org/doc/html/rfc8693#section-4.3)     |
+| `jti`        | REQUIRED          | Unique ID of this JWT.                                                                                                         | [Section 4.1.7 of RFC7519](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.7) |
+| `exp`        | REQUIRED          | Expiration time of this JWT (Unix timestamp in seconds).                                                                       | [Section 4.1.4 of RFC7519](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4) |
+| `iat`        | REQUIRED          | Issued-at time of this JWT (Unix timestamp in seconds).                                                                        | [Section 4.1.6 of RFC7519](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.6) |
+| `scope`      | OPTIONAL          | A JSON string containing a space-separated list of scopes associated with the token, per Section 3.3 of RFC6749.               | `["scope1 scope2"]`<br/>[Section 3.3 of RFC6749](https://datatracker.ietf.org/doc/html/rfc6749#section-3.3)    |
+
+The typ of the JWT indicated in the JWT header MUST be `oauth-id-jag+jwt`.
+
+An example JWT shown with expanded header and payload claims may look like this:
+
+```
+{
+  "typ": "oauth-id-jag+jwt"
+}
+.
+{
+  "jti": "9e43f81b64a33f20116179",
+  "iss": "https://acme.idp.example",
+  "sub": "U019488227",
+  "aud": "https://acme.chat.example/",
+  "client_id": "f53f191f9311af35",
+  "exp": 1311281970,
+  "iat": 1311280970,
+  "scope": "chat.read chat.history"
+}
+.
+signature
+```
+
+The authorization server **MAY** add additional claims as necessary.
+
+#### Implementation notes
+
+If the IdP is multi-tenant and uses the same issuer for all tenants, the MCP Server will already have IdP-specific logic to determine the tenant from the OpenID Connect ID Token (e.g., the `hd` claim for Google) or SAML assertion, and will need to use that if the IdP also has only one client registration for the MCP Server.
+
+`sub` should be an opaque ID, as `iss+sub` is unique. The IdP might want to also include the user's email here, which it should do as a new `email` claim. This would let the app dedupe existing users who may have an account with an email address but have not done SSO yet.
```

Comment:
> not done SSO yet

I think this could be confusing. Maybe something like:

```suggestion
`sub` should be an opaque ID, as `iss+sub` is unique. The IdP might want to also include additional user information, such as an email address, which it should do as a new `email` claim. This may allow the MCP Client application to properly link existing user accounts to the `sub` identifier used within the enterprise context for SSO.
```